--- RM	2015-07-12 17:44:48.936738602 +0000
+++ RM.new	2015-07-12 17:44:11.908234346 +0000
@@ -10,7 +10,7 @@
 # ORACLE 7
 #Oracle_XA:xaosw:-L${ORACLE_HOME}/lib -lxa ${ORACLE_HOME}/lib/libsql.a -lora -lsqlnet -lcore -lnlsrtl -lcore ${ORACLE_HOME}/lib/osntab.o -lsqlnet -lcv6 -lora -lm -lsocket
 # Oracle 9, 10
-Oracle_XA:xaosw:-L${ORACLE_HOME}/lib -lclntsh
+Oracle_XA:xaosw:-L/usr/lib/oracle/12.1/client64/lib -lclntsh
 
 # INFORMIX-Online 5
 INFORMIX-OnLine:infx_xa_switch:${INFORMIXDIR}/lib/esql/libinfxxa.a -lx
